src/serde/value/borrowed/se.rs

```diff
@@ -529,9 +529,7 @@ impl<'a> serde_ext::Serializer for MapKeySerializer<'a> {
 
     #[inline]
     fn serialize_str(self, value: &str) -> Result<Self::Ok> {
-        // TODO: This is ugly
-        //let s = value.to_owned();
-        Ok(value)
+        Ok(unsafe { std::mem::transmute(value) }) // TODO find a better way to serialize &str
```

Comment:
We need to find a way around this, this will give a segfault if str isn't living as long as `'se`

Reply:
We could try to return a `Cow` here instead of a `&str` that way we can just use `Cow::from(str.to_string())`

Reply:
Or even `String` perhaps?